uaclient/defaults.py

```diff
@@ -6,9 +6,12 @@
 """
 
 UAC_ETC_PATH = "/etc/ubuntu-advantage/"
+UAC_VAR_PATH = "/var/lib/ubuntu-advantage/"
```

Comment:
you probably also want to set the following in config defaults
"data_dir": UAC_VAR_PATH.rstrip("/")